src/sync/SSEHandler/NotificationKeeper.js

```diff
@@ -0,0 +1,56 @@
+import { PUSH_CONNECT, PUSH_DISCONNECT, PUSH_DISABLED, ControlTypes } from '../constants';
+
+const controlPriMatcher = /control_pri$/;
+
+export default function notificationKeeperFactory(feedbackLoopEmitter) {
+
+  let isStreamingUp;
```

Comment:
Start it with false. It's usually not great to change the type of the variable (reduces applicable machine code optimizations besides making it more confusing), even if JS allows for it. 
Use undefined for when you need to check if something is not set/defined, not only as a falsey value. For falsiness go with the boolean `false`.